Artifacts/Download-Artifacts.ps1

```diff
@@ -129,36 +129,39 @@ try {
                 }
```

Comment:
In the above section it mentions downloading application artifact and unpacking application artifact.
Remove the word "application" as it now can be core or platform